src/kakarot/instructions/environmental_information.cairo

```diff
@@ -424,16 +424,18 @@ namespace EnvironmentalInformation {
         // 2 - offset: byte offset in the code to copy.
         // 3 - size: byte size to copy.
         let (stack, popped) = Stack.pop_n(self=stack, n=4);
-        let address = popped[0];
+        let address_ = popped[0];
```

Comment:
would not block a PR for this but I'd like to have `address_uint256` & `address_felt` or something explicit like this when the two versions are used in the same piece of code. `address_` is less explicit

Reply:
makes sense ⚡